src/Main.js

```diff
@@ -16,6 +16,7 @@ export { default as FeatureProcessing } from 'Process/FeatureProcessing';
 export { updateLayeredMaterialNodeImagery, updateLayeredMaterialNodeElevation } from 'Process/LayeredMaterialNodeProcessing';
 export { default as OrientedImageCamera } from 'Renderer/OrientedImageCamera';
 export { default as PointsMaterial } from 'Renderer/PointsMaterial';
+export { default as GlobeControls } from 'Controls/GlobeControls';
```

Comment:
You can keep 1 commit for this pull request.
But it's better to separate by scope, the modifications in different commits.